nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/dependencies/TargetPlatformFilterHelper.java

```diff
@@ -20,23 +20,46 @@
 import org.apache.commons.io.FilenameUtils;
 import org.apache.commons.io.IOCase;
 import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.nuxeo.connect.packages.dependencies.versioning.PlatformVersion;
+import org.nuxeo.connect.packages.dependencies.versioning.PlatformVersionRange;
 import org.nuxeo.connect.update.Package;
 
 /**
  * @author <a href="mailto:td@nuxeo.com">Thierry Delprat</a>
  */
 public class TargetPlatformFilterHelper {
 
-    public static boolean isCompatibleWithTargetPlatform(Package pkg, String targetPlatform) {
-        return isCompatibleWithTargetPlatform(pkg.getTargetPlatforms(), targetPlatform);
+    protected static Log log = LogFactory.getLog(TargetPlatformFilterHelper.class);
+
+    public static boolean isCompatibleWithTargetPlatform(Package pkg, String targetPlatform,
+            String targetPlatformVersion) {
+        String tpRangeSpec = pkg.getTargetPlatformRange();
+        if (StringUtils.isBlank(pkg.getTargetPlatformRange()) || targetPlatformVersion == null) {
```

Comment:
Related to `https://github.com/nuxeo/nuxeo-connect/pull/35/files#diff-00253ed9badc2f8396c815044fb2165fR106`, I think it's safer to fallback on backward compat' if `targetPlatformVersion` is blank too

Reply:
It will fallback if there is a parsing error + log a warning. It was a choice to separate null (coming from old platforms) and blank (coming from a problem on new platforms)

Reply:
But the log is not correct ;)